content/python/concepts/random-module/random-module.md

```diff
@@ -1,151 +1,27 @@
 ---
 Title: 'Random Module'
-Description: 'Python has a special module called random that can be used to randomly generate or modify integers, floats, lists, and strings. In order to use the random module, import it at the top of a Python file: py import random  For example, the .random() method will generate a float greater than or equal to 0.0 and less then 1.0. codebyte/py'
+Description: 'Python has a random module that is used for working with randomly-generated numbers.'
 Subjects:
   - 'Data Science'
   - 'Computer Science'
 Tags:
+  - 'Modules'
   - 'Random'
-  - 'Integers'
-  - 'Strings'
-  - 'Methods'
-  - 'Lists'
 CatalogContent:
   - 'learn-python-3'
   - 'paths/computer-science'
 ---
 
-Python has a special module called [random](https://docs.python.org/3/library/random.html) that can be used to randomly generate or modify integers, floats, lists, and strings.
+The [`random` module](https://docs.python.org/3/library/random.html) is built into Python and used for working with numbers randomly generated from collections such as [lists](https://www.codecademy.com/resources/docs/python/lists) and [strings](https://www.codecademy.com/resources/docs/python/strings).
```

Comment:
```suggestion
The [`random` module](https://docs.python.org/3/library/random.html) is built into Python and used for working with randomly generated numbers, and randomly manipulate collections such as [lists](https://www.codecademy.com/resources/docs/python/lists) and [strings](https://www.codecademy.com/resources/docs/python/strings).
```

Reply:
@SSwiniarski Great suggestion! What do we think of this for a final edit (for this review)?

```
In Python, the built-in [`random` module] is used for randomly generating numbers as well as randomly manipulating collections such as [lists] and [strings].
```

Reply:
@Dusch4593, that looks good to me.